website/src/site/styles/layout/content.scss

```diff
@@ -26,24 +26,7 @@ $transition-timing: 300ms;
   }
 }
 
-.ds-subpage-section {
-  display: grid;
-  grid-column-gap: 7.2rem;
-  grid-template-columns: 1fr;
-  margin: 2.4rem 0;
-
-  @media screen and (min-width: $xl-breakpoint) {
-    grid-template-columns: 6fr 1fr;
-  }
-
-  @media screen and (min-width: $xxl-breakpoint) {
-    grid-template-columns: 88rem 1fr;
-  }
-
-  h2 {
-    margin-top: 2.4rem;
-  }
-
+*:where(.ds-subpage-section, .ds-tabs__tab-container) {
```

Comment:
One minor thing, this `header-anchor` should not have text-decoration as well :) actually all `header-anchor` elements should have no text-decoration:

<img width="895" alt="image" src="https://github.com/sl-design-system/components/assets/111562742/af74b766-f56c-49fa-80be-082a04cbf113">

Reply:
So these ones as well (all `header-anchor` elements):

<img width="878" alt="image" src="https://github.com/sl-design-system/components/assets/111562742/124c2dcd-8618-4a0d-808a-560e437152e8">